app/errors/address_lookup_error.rb

```diff
@@ -1,5 +1,5 @@
 class AddressLookupError < StandardError
-  def initialize(message)
-    super(message)
+  def initialize(code)
+    super(I18n.t("coronavirus_form.api_errors.#{code}"))
```

Comment:
It would be worth having a default here, since it could be that you receive a different status code, or `nil` as the code.

Reply:
Have added in a default value :)